apis/apps/v1alpha1/cluster_types.go

```diff
@@ -312,14 +314,14 @@ type ComponentMessageMap map[string]string
 // ClusterComponentStatus records components status.
 type ClusterComponentStatus struct {
 	// phase describes the phase of the component and the detail information of the phases are as following:
-	// Running: the component is running. [terminal state]
-	// Stopped: the component is stopped, as no running pod. [terminal state]
-	// Failed: the component is unavailable, i.e. all pods are not ready for Stateless/Stateful component and
-	// Leader/Primary pod is not ready for Consensus/Replication component. [terminal state]
-	// Abnormal: the component is running but part of its pods are not ready.
-	// Leader/Primary pod is ready for Consensus/Replication component. [terminal state]
 	// Creating: the component has entered creating process.
+	// Running: the component is running.
 	// Updating: the component has entered updating process, triggered by Spec. updated.
+	// Stopping: the component is stopping.
+	// Stopped: the component is stopped, as no running pod.
+	// Deleting: the component is being deleted.
+	// Failed: the component is unavailable, i.e. all pods are not ready.
```

Comment:
if any pod of the component is not ready, component phase will be Failed?

Reply:
has reverted `Failed` to the original meaning.